tests/Neo.UnitTests/Ledger/UT_TransactionState.cs

```diff
@@ -52,27 +53,75 @@ public void Initialize()
         [TestMethod]
         public void TestDeserialize()
         {
-            var data = BinarySerializer.Serialize(((IInteroperable)origin).ToStackItem(null), ExecutionEngineLimits.Default);
+            var data = BinarySerializer.Serialize(((IInteroperable)_origin).ToStackItem(null), ExecutionEngineLimits.Default);
             var reader = new MemoryReader(data);
 
             TransactionState dest = new();
             ((IInteroperable)dest).FromStackItem(BinarySerializer.Deserialize(ref reader, ExecutionEngineLimits.Default, null));
 
-            Assert.AreEqual(origin.BlockIndex, dest.BlockIndex);
-            Assert.AreEqual(origin.Transaction.Hash, dest.Transaction.Hash);
+            Assert.AreEqual(_origin.BlockIndex, dest.BlockIndex);
+            Assert.AreEqual(_origin.Transaction.Hash, dest.Transaction.Hash);
             Assert.IsNotNull(dest.Transaction);
         }
 
+        [TestMethod]
+        public void TestClone()
+        {
+            var clone = ((IInteroperable)_origin).Clone() as TransactionState;
+            CollectionAssert.AreEqual(
+                BinarySerializer.Serialize((clone as IInteroperable).ToStackItem(null), ExecutionEngineLimits.Default),
+                BinarySerializer.Serialize((_origin as IInteroperable).ToStackItem(null), ExecutionEngineLimits.Default)
+                );
+            clone.Transaction.Nonce++;
+            Assert.AreNotEqual(clone.Transaction.Nonce, _origin.Transaction.Nonce);
+        }
+
+        [TestMethod]
+        public void AvoidReplicaBug()
```

Comment:
This test fault before this pr, multiple `FromReplica` to same object use a different `_rawTransaction`.